gapic-generator-java/src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -598,9 +600,16 @@ private static Map<String, Message> parseMessages(
         }
       }
     }
+
+    ImmutableList<String> nestedEnums =
+        messageDescriptor.getEnumTypes().stream()
```

Comment:
I guess `getEnumTypes()` is empty if a message does not have any nested enums?

Reply:
Correct. Most messages don't have any nested enums.